eclipse-cdt-cloud/cdt-cloud-blueprint
# Add support for bundling trace server

#### What it does

1. Package the trace server in the CDTCloudBlueprint.AppImage for Linux (fixes #17)
2. Support running CDT cloud blueprint from development environment (both browser and electron app). Set trace server path for that.

#### How to test

1. Run `yarn tracecompass-server:download` first and then package the application image on Linux. Start the application and use command `Start trace server` to start trace server. The server should start successfully.
2. `yarn browser` and connect with browser to localhost:3000 or `yarn electron`. In the running application use command `Start trace server` to start trace server. The server should start successfully.

Note, that this PR also includes a commit to Workaround rate limitation issues with open-vsx. It downloads the plug-ins sequentially instead of parallel. This should avoid the failure "Download:plugins fails with 429". See discussion:
https://community.theia-ide.org/t/download-plugins-fails-with-429/2659
